sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -1774,7 +1775,7 @@ void captureSessionToken(RxDocumentServiceRequest request, RxDocumentServiceResp
         final Map<String, String> requestHeaders = getRequestHeaders(options, ResourceType.Document, OperationType.Patch);
         Instant serializationStartTimeUTC = Instant.now();
 
-        ByteBuffer content = ByteBuffer.wrap(PatchUtil.serializeCosmosPatchToByteArray(cosmosPatchOperations));
+        ByteBuffer content = ByteBuffer.wrap(PatchUtil.serializeCosmosPatchToByteArray(new PatchSpec(cosmosPatchOperations, options)));
```

Comment:
Is it necessary to pass patch spec to function? Since this is just an utility class? Creating a new instance just for function input doesn't make sense to me.

Reply:
I also don't see why we need that.